app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -597,30 +647,65 @@ export default class Heatmap {
     this.metadataLabelsBackground.attr("x", x - this.options.marginLeft);
   }
 
-  placeColumnLabelAndMetadataContainers(y) {
-    this.gColumnLabels.attr(
-      "transform",
-      `translate(${this.rowLabelsWidth},${y})`
-    );
-    this.gColumnMetadata.attr("transform", `translate(0, ${y})`);
-    // Placing the white background rectangle behind the column labels and metadata.
-    this.columnLabelsBackground.attr(
-      "y",
-      y - this.columnLabelsHeight - this.options.marginTop
-    );
-    // Placing the white rectangle to hide column labels in the top left corner.
-    this.metadataLabelsBackground.attr(
-      "y",
-      y - this.columnLabelsHeight - this.options.marginTop
-    );
+  placeColumnLabelAndMetadataContainers(y, transition = false) {
+    if (transition) {
```

Comment:
Sorry, i am having trouble understanding what changes in column label and metadata positioning.  Can you explain the difference?

Reply:
There were no changes in the column label and metadata positioning, I just added a transition so that the heatmap will smoothly scroll to the row instead of jumping to it. This normally isn't necessary when the user manually scrolls since the the delta from scrolling is much more incremental.

Reply:
Ok, would just set duration to 0 in case of no transition work?

Reply:
Ah yes that works as well. Just changed!